structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/cdp/CDPStructuredFlowEvent.java

```diff
@@ -0,0 +1,73 @@
+package com.sequenceiq.cloudbreak.structuredevent.event.cdp;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.sequenceiq.cloudbreak.structuredevent.event.FlowDetails;
+import com.sequenceiq.cloudbreak.structuredevent.json.Base64Deserializer;
+import com.sequenceiq.cloudbreak.structuredevent.json.Base64Serializer;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CDPStructuredFlowEvent<T> extends CDPStructuredEvent {
+    private FlowDetails flow;
+
+    @JsonSerialize(using = Base64Serializer.class)
+    @JsonDeserialize(using = Base64Deserializer.class)
+    private String exception;
+
+    private T payload;
+
+    public CDPStructuredFlowEvent() {
+        super(CDPStructuredFlowEvent.class.getSimpleName());
+    }
+
+    public CDPStructuredFlowEvent(String type, CDPOperationDetails operation, FlowDetails flow, T payload) {
+        super(type, operation);
+        this.flow = flow;
+        this.payload = payload;
+    }
+
+    @Override
+    public String getStatus() {
+        String state = flow.getFlowState();
+//        if ("unknown".equals(state)) {
+//            state = getOperation().getResourceType().toUpperCase()
+//            if (stack != null) {
+//                state = "STACK_" + stack.getDetailedStatus();
+//            }
+//            if (cluster != null) {
+//                state += " | CLUSTER_" + cluster.getStatus();
+//            }
+//        }
```

Comment:
do we need these or could be deleted? if can be then you should return in the first line and you don't need the state variaable